diff --git a/lib/publish.js b/lib/publish.js
index 3d97124963becac6535aed3ad31bb236dc3c2f51..9b81a02fbf4192a5217c1eab77e9e32b68c2c4b8 100644
--- a/lib/publish.js
+++ b/lib/publish.js
@@ -19,10 +19,14 @@ module.exports = async (npmrc, {npmPublish, pkgRoot}, pkg, context) => {
     const registry = getRegistry(pkg, context);
     const distTag = getChannel(channel);
 
+    logger.log('Kirill has been here');
     logger.log(`Publishing version ${version} to npm registry on dist-tag ${distTag}`);
     const result = execa(
       'pnpm',
-      ['publish', basePath, '--userconfig', npmrc, '--tag', distTag, '--registry', registry],
+      ['publish', basePath,
+        // '--userconfig', npmrc,
+        '--no-git-checks',
+        '--tag', distTag, '--registry', registry],
       {cwd, env, preferLocal: true}
     );
     result.stdout.pipe(stdout, {end: false});
